tsc/src/audio/audio.hpp

```diff
@@ -24,6 +24,9 @@
 #include "../scripting/scriptable_object.hpp"
 #include "../scripting/objects/misc/mrb_audio.hpp"
 #include <queue>
+#undef IN
+#undef OUT
```

Comment:
Won't this print warnings if `IN` and `OUT` aren't defined? (Also I'm going to guess this is from `Windows.h` *sigh*...)

Reply:
Suggestion: Add a comment on why this is there in the first place (it's not self-explanatory). Also, you might want to consider wrapping it up with an `#ifdef _WIN32`.

And yes, I suspect `Windows.h` as well.

Reply:
The error I received was "Unrecognized [something] before `','` on line 193. I will add the win32 check right now.